tests/model_forms/test_modelchoicefield.py

```diff
@@ -190,6 +204,12 @@ def test_disabled_modelchoicefield_has_changed(self):
         field = forms.ModelChoiceField(Author.objects.all(), disabled=True)
         self.assertIs(field.has_changed('x', 'y'), False)
 
+    def test_disabled_and_instance_initial_modelchoicefield(self):
```

Comment:
Can we adjust the test name. 

We know this is `modelchoicefield`, because the whole `TestCase` is called that, so we can drop that. 

Maybe... `test_initial_accepts_model_instance_for_validation_when_field_disabled`? It's a bit long and horrible but... (???: suggestions welcome!)

Reply:
Commit updated.
No better idea. Should we add ticket ref for more details ?